tensorflow/lite/g3doc/apis.md

```diff
@@ -9,7 +9,7 @@ convenience.  Similarly, consistency with TensorFlow APIs was not an explicit
 goal and some variance is to be expected.
 
 There is also a Python API for TensorFlow Lite described
-[here](../toco/g3doc/python_api.md#interpreter).
```

Comment:
Given that this is a document about the TFLite interpreter APIs, `#interpreter` seems correct.

To clarify, can you change "TensorFlow Lite described" --> "TensorFlow Lite interpreter, described".

Thanks for making this fix.

Reply:
@gargn 
Thank you for review.
I think you are right.
I fixed the referenced anchor.
Can you please review again?
**From**
```
#converter-python-api-guide
```
**To**
```
#tensorflow-lite-python-interpreter-
```

Reply:
To clarify what I meant, can you make your change the following:

There is also a Python API for TensorFlow Lite interpreter described [here](convert/python_api.md#interpreter).

Reply:
@gargn 
Sorry for the inconvenience to you.
Your suggestion:
https://github.com/tensorflow/tensorflow/pull/convert/python_api.md#interpreter
![bbb](https://user-images.githubusercontent.com/33194443/48590724-f5492800-e983-11e8-81ad-208163fa525d.png)
And, And in your previous proposal **`../toco/g3doc/python_api.md`** there is no document in the current **`master`** branch.
It appears to have been deleted 3 days ago.
![aaa](https://user-images.githubusercontent.com/33194443/48590777-33464c00-e984-11e8-8f95-bcc4b07e8f87.png)
And the current **`tensorflow/lite/g3doc/convert/python_api.md`** does not have "#interpreter" anchor.
I wonder how to use Github. . .
I am a little confused. . .